### PR TITLE
sample

### PR DESCRIPTION
**Pull request by omaraboumrad**
added foo for test

### CODE REVIEW

**Comment by omaraboumrad** [comment]
some comment

**Comment by joehakimrahme** [comment]
Hmmm can you plz add some text in the file so we can test inline comments too?

File: foo.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Added somestuff
+with some more stuff
+
+and then some
```
Comment (joehakimrahme) [inline foo.txt:6]: Here's an inline comment. Push a new version and let's see if the inline survives

Comment (omaraboumrad) [inline foo.txt:6]: ok

**Pull request closed without merging**